decomposition.tex

```diff
@@ -137,21 +137,21 @@
     // addressability
     { &t } -> Same<T*>;
```

Comment:
Correction: all two variants. I don't think we should to require `&` for rvalues.